lazy_tensor_core/third_party/computation_client/metrics.cc

```diff
@@ -286,23 +285,24 @@ std::string MetricFnBytes(double value) {
 }
 
 std::string MetricFnTime(double value) {
+  const int kNumUnits = 6;
```

Comment:
`sizeof` doesn't work here?

Reply:
Oh, it does. But that `TF_ARRAYSIZE` macros is only used in this function, so I decided to just get rid of it.